addons/binding/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/KonnectedConfiguration.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.konnected.internal;
+
+/**
+ * The {@link KonnectedConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Zachary Christiansen - Initial contribution
+ */
+public class KonnectedConfiguration {
+
+    /**
+     * Sample configuration parameter. Replace with your own.
+     */
+    public String Auth_Token;
```

Comment:
Use camel casing with "Auth_Token" e.g. "authToken" and update/remove the javadoc.